javascript/javascript1/week4/Voice-Assistant.js

```diff
@@ -0,0 +1,86 @@
+let vname = '';
+const toDo = [];
+function getReply(command) {
+  const arrayOfStrings = command.split(" ");
+  
+  if (command.startsWith("Hello my name is")) {
+    vname = arrayOfStrings[4];
+    return `Nice to meet you ${vname}.` ;
+  }
+  if (command =="What is my name?") {
```

Comment:
Try not to use `==` ever. Even if the two sides of this are of different data types then use some parser, but always use `===`.

Reply:
done .. thanks